Formula/healpix.rb

```diff
@@ -0,0 +1,53 @@
+class Healpix < Formula
+  desc "Hierarchical Equal Area isoLatitude Pixelization of a sphere"
+  homepage "https://healpix.jpl.nasa.gov"
+  url "https://downloads.sourceforge.net/project/healpix/Healpix_3.40/Healpix_3.40_2018Jun22.tar.gz"
+  version "3.40"
+  sha256 "f10ce170a10a2f37830c65616554c39005442021741ed19c1efa998994d8a069"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cfitsio"
+
+  def install
+    configure_args = %W[  --disable-dependency-tracking
+                          --disable-silent-rules
+                          --prefix=#{prefix}  ]
+
+    cd "src/C/autotools" do
+      system "autoreconf", "--install"
+      system "./configure", *configure_args
+      system "make", "install"
+    end
+
+    ENV.append_to_cflags "-DENABLE_FITSIO"
+    cd "src/cxx/autotools" do
+      system "echo", "\"AUTOMAKE_OPTIONS = subdir-objects\" >> Makefile.am"
```

Comment:
Why do we need to ad this to `Makefile.am`?

Reply:
It was unnecessary, Removed